packages/keyring-eth-hd/test/index.test.ts

```diff
@@ -154,7 +159,7 @@ describe('hd-keyring', () => {
         numberOfAccounts: 2,
       });
 
-      const accounts = await keyring.getAccounts();
+      const accounts = keyring.getAccounts();
```

Comment:
`getAccounts()` was awaited everywhere in the tests, but the function implementation was not marked as `async`

Reply:
we can change the tests here, and then change them back along with fixing the implementation when `HDKeyring` will implement the `Keyring` type